rel/overlay/etc/default.ini

```diff
@@ -24,6 +24,10 @@ file_compression = snappy
 ; and/or more OS page cache hits, but they can also increase overall response
 ; time for writes when there are many attachment write requests in parallel.
 attachment_stream_buffer_size = 4096
+; Default security object for databases if not explicitly set
+; open - same as couchdb 1.0, everyone can read/write
+; admin_only - only admins can read/write
+default_security = open
```

Comment:
I like the naming, I think the natural opposites here are:

- `open` / `closed`
- `everyone` / `admin_only`

and since “closed” isn’t very descriptive I think `everyone` / `admin_only` is my preference